styles/components/app-header.css

```diff
@@ -0,0 +1,12 @@
+.quiz-app-heading {
```

Comment:
When i understood right, teh quiz- needs to be removed as all classes need to start with the document name. so app-header 😊

Reply:
you are right 😁 many thanks for - probably i will change the file name, so that it will fit again